src/fu-util.c

```diff
@@ -957,17 +957,36 @@ fu_util_get_history (FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static FwupdDevice*
+fu_util_get_device_or_prompt (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	FwupdDevice *dev = NULL;
+
+	/* get device to use */
+	if (g_strv_length (values) >= 1) {
+		g_autoptr(GError) error_local = NULL;
+		if (g_strv_length (values) > 1)
```

Comment:
I think you need `{``s around  this if - it's multiline and so a bit confusing.

Reply:
I'll double check, but even if it works style wise you'd rather they be there?

Reply:
it just looked odd to me to see a "single line" if with a for loop nested below.